src/services/sendMessage.ts

```diff
@@ -1,28 +1,29 @@
-import requestPromise from 'request-promise';
-
 import { Soldier, Message, Cookie, SoldierClass } from '../models';
-import { buildRequestUrl, addLog } from '../utils';
+import { buildRequestUrl, addLog, request, RequestOption } from '../utils';
 
 /**
  * 인터넷 편지를 전송한다.
  * @param cookies - 세션 식별을 위한 쿠키
  * @param trainee - 훈련병 정보
  * @param message - 인터넷 편지 정보
  */
-async function sendMessage(cookies: Cookie, trainee: Soldier, message: Message) {
+async function sendMessage(
+  cookies: Cookie,
+  trainee: Soldier,
+  message: Message
```

Comment:
```suggestion
  message: Message,
```
trailing comma가 필요합니다!

Reply:
https://github.com/parksb/the-camp-lib/pull/16/commits/64f14bccf048398a596cd98bcc33b0237240bcb9 반영하였습니다.